src/bci_build/package/mariadb/README.server.md.j2

```diff
@@ -0,0 +1,66 @@
+# {{ image.pretty_name }} Container Image
+
+MariaDB is one of the most popular database servers in the world. It's made by the original developers of MySQL and guaranteed to stay open source.
+
+## How to use this image
+
+By default, the image launches MariaDB with the same configuration as the one
+that comes with SUSE Linux Enterprise Server. The differences are that logging is
+sent to stdout and binding to `localhost` is disabled.
+
+The only environment variable that is required to start the container is the MariaDB root password.
+
+```ShellSession
+$ podman run -it --rm -p 3306:3306 -e MARIADB_ROOT_PASSWORD=my-password registry.suse.com/suse/{{ image.name }}:{% if image.is_opensuse %}latest{% else %}{{ image.version }}{% endif %}
```

Comment:
no, opensuse images are not on registry.suse.com. I think there is a full variable that has the actual path. if there isn't then we should add that.

Reply:
```suggestion
$ podman run -it --rm -p 3306:3306 -e MARIADB_ROOT_PASSWORD=my-password {{ image.reference | replace("-%RELEASE%") }}
```

Reply:
Fixed